packages/javascript/bh-shared-ui/src/components/GraphButton/GraphButton.tsx

```
@@ -40,17 +40,20 @@ const useStyles = makeStyles((theme) => ({
-    onClick: (e?: any) => void;
+export interface GraphButtonProps extends ButtonProps {
     displayText: string | ReactNode;
+    onClick: (e?: any) => void;
     disabled?: boolean;
 }
 
-const GraphButton: FC<GraphButtonProps> = ({ onClick, displayText, disabled }) => {
+const GraphButton: FC<GraphButtonProps> = (props) => {
+    const { displayText } = props;
+    const attributes = { ...props };
+    delete attributes.displayText;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)